sros2/setup.py

```diff
@@ -11,17 +11,21 @@ def package_files(directory):
             paths.append(os.path.join('..', path, filename))
     return paths
 
-
```

Comment:
The linter is complaining about this change.

Reply:
@jacobperron , I've reverted this line.